misc/jenkins/checkwhitespaces.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+# This file is part of NIT ( http://www.nitlanguage.org ).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Check whitespace errors in commits
+# Usage: checkwhitespaces from to
+#
+# This script is in fact a more friendly version of `git log --check`
+
+set -e
+
+from=${1:-origin/master}
+to=${2:-HEAD}
+
+err=0
+
+cd `git rev-parse --show-toplevel`
+
+echo "checkwhitespaces $from (`git rev-parse "$from"`) .. $to (`git rev-parse "$to"`)"
+for ref in `git rev-list --no-merges "$from".."$to"`; do
+	# Show nothing if no error
+	if git --no-pager show --check --oneline $ref > /dev/null; then
+		continue
+	fi
+
+	# Run the command again to display things
+	echo ""
+	echo "Found whitespace errors for commit"
+	git --no-pager show --check --oneline $ref || true
+	err=1
+done
+
+if test "$err" = 1; then
+	echo ""
+	echo "Please check that each file in each commit does not contains whitespace errors."
```

Comment:
Yes, it is displayed in the message.
As an example, `./checkwhitespaces.sh HEAD e6cfa93`, the output is

~~~
Found whitespace errors for commit
e6cfa93 Started documenting the digraph module.
lib/graphs/digraph.nit:5: trailing whitespace.
+# 
lib/graphs/digraph.nit:13: trailing whitespace.
+# 
lib/graphs/digraph.nit:18: trailing whitespace.
+# 
lib/graphs/digraph.nit:21: trailing whitespace.
+# 

Found whitespace errors for commit
e5c4311 Moved modules to lib/graphs.
lib/graphs/digraph.nit:313: trailing whitespace.
+		for 

Found whitespace errors for commit
5e7bf24 Started MatrixDigraph.
lib/digraph.nit:313: trailing whitespace.
+		for 

Found whitespace errors for commit
e6fd4d4 Basic methods for digraphs.
tests/test_graph.nit:34: new blank line at EOF.

Please check that each file in each commit does not contain whitespace errors.
Note that existing commits should be amended; pushing new commit is not sufficient.
Hint: use "git log --check" to see whitespace errors.
~~~

So you can check if the wording is informative enough. If you have a proposition to improve the output I will include it.